tests/ui/conftest.py

```diff
@@ -58,3 +60,33 @@ def user(base_url, fxa_account, jwt_token):
     assert requests.codes.created == r.status_code
     user.update(r.json())
     return user
+
+
+@pytest.yield_fixture(scope='session')
+def firefox_path(tmpdir_factory):
```

Comment:
So add that as a parameter with tmpdir_factory, in doing so, that will call the normal fixture to see if it returns anything, and then it doesn't we use the downloaded version?

Perhaps we should figure out a way to create a marker for our discopane tests to override the firefox_path. Even if a firefox_path is specified, we still may need a different version for discopane, right?